pkg/kubelet/cm/container_manager.go

```diff
@@ -49,7 +49,7 @@ import (
 
 const (
 	// Warning message for the users still using cgroup v1
-	CgroupV1MaintenanceModeWarning = "cgroup v1 support is in maintenance mode, please migrate to cgroup v2"
+	CgroupV1UnsupportedWarning = "cgroup v1 support is unsupported and will be removed in a future release. Please migrate to cgroup v2. More information at https://git.k8s.io/enhancements/keps/sig-node/5573-cgroup-v1-unsupported"
```

Comment:
The warning message is not clear to me. It said v1 is unsupported but will be removed.

`cgroup v1 support is no longer maintained and will be removed in an upcoming release.` or `cgroup v1 support is deprecated and will be removed in a future release`. 

This may need to de decided after the KEP was merged.